pkg/resources/manifests/deploy.yaml

```diff
@@ -176,6 +176,10 @@ spec:
           - name: HOST_ROOT
             value: "/host"
         securityContext:
+          # We need this to be able mount host directories with correct permissions on OpenShift
+          # The option is ignored for non SELinux systems.
```

Comment:
Maybe you can add the cri-o link in a comment somewhere:
https://github.com/cri-o/cri-o/blob/v1.27.0/server/sandbox_run_linux.go#L537

Reply:
Done. I updated the comment to include more context!